python/ray/data/tests/test_consumption.py

```diff
@@ -249,6 +249,12 @@ def test_schema_lazy(ray_start_regular_shared):
     assert ds._plan.execute()._num_computed() == 0
 
 
+def test_columns(ray_start_regular_shared):
+    ds = ray.data.range(100, parallelism=10)
```

Comment:
Nit: Any reason we need the `parallelism=10`?
```suggestion
    ds = ray.data.range(1)
```

Writing minimally passing tests is a best practice.